src/main/java/io/citrine/jcc/search/pif/result/PifMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.pif.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a PIF multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class PifMultiSearchResultElement extends BaseMultiSearchResultElement<PifSearchResult> {
```

Comment:
Generally no. But it makes sense in this case. The query that is executed defines the type of the generic. There is no reason for the user to have to supply that information since it's already determined. And there are only two implementations that we have to define.